internal/impl/devbox.go

```diff
@@ -362,15 +363,18 @@ func (d *Devbox) GenerateEnvrc(force bool, source string) error {
 	if commandExists("direnv") {
 		// prompt for direnv allow
 		var result string
-		prompt := &survey.Input{
-			Message: "Do you want to enable direnv integration for this devbox project? [y/N]",
-		}
-		err := survey.AskOne(prompt, &result)
-		if err != nil {
-			return errors.WithStack(err)
+
+		if isatty.IsTerminal(os.Stdin.Fd()) {
```

Comment:
Could we make the result a bool outside of the `if` so we only need to do `isatty` once? Something like:

```go
enableDirEnv := true
if isatty.IsTerminal(os.Stdin.Fd()) {
	// ...
	enableDirEnv := strings.ToLower(result) == "y"
}

if enableDirEnv {
	// ...
}
```